qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -7,3 +7,8 @@
     Linux:
         x86_64:
             check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
+        aarch64:
+            no RHEL.6 RHEL.7 RHEL.8
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            required_qemu = [6.2.0,)
+            check_PPTT_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
```

Comment:
So, PPTT is not a property, for this checkpoint seems to check the core ID mapping.

Reply:
Thanks for the reminder, corrected.